include/mlir/Dialect/StandardOps/Ops.td

```diff
@@ -881,6 +881,14 @@ def ShlISOp : IntArithmeticOp<"shlis"> {
   let summary = "signed integer shift left";
 }
 
+def ShrISOp : IntArithmeticOp<"shris"> {
+  let summary = "signed integer shift right";
+}
+
+def ShrIUOp : IntArithmeticOp<"shriu"> {
+  let summary = "unsigned integer shift right";
+}
+
```

Comment:
I'm wondering where is this naming coming from? Is there some kind of standard?
I find these fairly hard to read if you're not using them daily (the LLVM ones as well by the way, but LLVM is more contained).
I wonder if we could read a custom printer/parser like:

```
%2 = bitshift(%0, %1)  {right, signed}  : i16
```

or more inventively:

```
%2 = signed<<(%0, %1) : i16
```

Reply:
There was already a `shlis` op in MLIR (it was added back when `addi` was introduced) so I just adopted that naming for the right shifts. Also this name seems to be consistent with `remis` and `diviu`.

That being said, I also like the `bitshift` syntax. We should just make sure that encoding the `signed`/`unsigned` behavior in an attribute is the right thing to do (because all other standard ops encode this in their name).

Reply:
One more question: Should we have a signed left shift at all? Wouldn't a generic left shift be sufficient (also there is only one integer add op for both signed and unsigned)?